### ISSUE TITLE
generate:cest sets incorrect Tester typehints when class is namespaced

### ISSUE THREAD

**Issue by Naktibalda** [Codeception/Codeception#4874]
#### What are you trying to achieve?

Generate valid test file

#### What do you get instead?

Incorrect FunctionalTester typehint

```sh
codecept g:cest functional Module/Create
Test was created in /.../tests/functional/Module/CreateCest.php
```

Generated file looks like this:
```php
<?php
namespace Module;


class CreateCest
{
    public function _before(FunctionalTester $I)
    {
    }

    public function _after(FunctionalTester $I)
    {
    }

    // tests
    public function tryToTest(FunctionalTester $I)
    {
    }
}
```

This issue can be fixed by changing typehint to fully qualified name \FunctionalTester
or by adding `use FunctionalTester;` between namespace and class;

I am using codeception 2.4.0

**Comment by andrew-womeldorf** [comment]
Yes, I have this same issue.  I have to manually fix the namespace of the generated cests each time I use the generator.

**Comment by Naktibalda** [comment]
Fixed in 5.0
